### DISCUSSION TITLE
Healthcheck "NOK"

### DISCUSSION THREAD

**Discussion by tm-wtnbg** [ZoeyVid/NPMplus#1111] [Q&A] [answered]
Hello,

the healtcheck is unfortunately always “NOK” because only “http” and no “https” is supported by the API or there is no response for “https”:

```
curl -sk https://127.0.0.1:81/api/
nothing returned
```
```
curl -sk http://127.0.0.1:81/api/
{"status":"OK","version":{"major":2,"minor":11,"revision":3}}
```

Best regards

Tim

**Comment by Zoey2936** [comment, marked as answer]
Hello, https is correct: https://github.com/ZoeyVid/NPMplus/blob/a8713607a58714fd132b4cf4e4a6389fa55a7ad6/rootfs/usr/local/nginx/conf/conf.d/npm.conf#L3
